packages/atomic/src/index.html

```diff
@@ -106,6 +107,68 @@
         margin: 10px 0;
       }
 
+      @media only screen and (max-width: 1024px) {
+        atomic-search-interface {
+          grid-template-rows: 16px 50px 16px auto 1fr;
+          grid-template-columns: minmax(0, 100%);
+        }
+
+        atomic-facet-manager,
+        .header,
+        atomic-search-box,
+        .results-container {
+          grid-column: 1 / auto;
+        }
+
+        atomic-breadcrumb-manager,
+        atomic-query-summary,
+        atomic-no-results,
+        atomic-did-you-mean,
+        atomic-query-error,
+        atomic-sort-dropdown,
+        atomic-facet-manager {
+          padding: 0 15px;
+        }
+
+        atomic-atomic-breadcrumb-manager,
+        atomic-query-summary,
+        atomic-no-results,
+        atomic-query-error,
+        atomic-sort-dropdown {
+          margin-bottom: 10px;
+        }
+
+        atomic-facet-manager {
+          grid-row: 4 / auto;
+          display: flex;
+        }
```

Comment:
I think the grid styles should use a CSS class, not the atomic-facet-manager as the selector, because some people may not want to use this component, or may use multiple instances of it.

It is similar to the `.results-container` below.

This could be something to revise at a later point though,

Reply:
Yes, absolutely. It's for the index.html / "default layout that we ship for v0".

But, as said above, how we decide to ship multiple layouts/theme is still un in the air and we will have to discuss this leading up to v1.